index.html

```diff
@@ -53,6 +53,8 @@
         <li><a href="#code">Code</a></li>
         <li><a href="#tables">Tables</a></li>
         <li><a href="#popovers">Popovers</a></li>
+        <li><a href="#cards">Cards</a></li>
```

Comment:
Looks like these two need the id="cards" and id="badges" for their section for the links to work.

Reply:
done

Reply:
done